spec/spec_helper.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 require 'vcr'
```

Comment:
Looks like we will need to do this: https://github.com/codacy/ruby-codacy-coverage#troubleshoot